content/tracing/_index.md

```diff
@@ -2,75 +2,204 @@
 title: APM (Tracing)
 kind: Documentation
 description: Instrument your code to improve performance
+further_reading:
+- link: "/tracing/setup"
+  tag: "Documentation"
+  text: Instrument your code
+- link: "/tracing/services"
+  tag: "Documentation"
+  text: Analyse your services
+- link: "/tracing/traces"
+  tag: "Documentation"
+  text: Analyse your traces
 ---
 
 ## Overview
 
 Datadog's integrated APM tool eliminates the traditional separation between infrastructure and application performance monitoring. This not only provides greater visibility, but also allows you to see the relationship between application code and the underlying infrastructure.
 
+Datadog APM is offered as an upgrade to our Pro and Enterprise plans. A free 14-day trial is available.
+Registered users can visit the [APM page of the Datadog application](https://app.datadoghq.com/apm/home) to get started.
+
 ## Getting started
 
-Datadog APM is offered as an upgrade to our Pro and Enterprise plans. A free 14-day trial is available.  Registered users can visit the [APM page of the Datadog application](https://app.datadoghq.com/apm/home) to get started.
+To understand how tracing work, let's take the following example that represent a simple API **thinker-api** and a micro-service behind it **thinker-microservice**, when the api receive a request with the correct *subject* parameter, it answer with a thought, otherwise it answer an error:
+
+{{< img src="tracing/tracing_overview_GS.jpg" alt="Tracing getting started overview" responsive="true" popup="true">}}
+
+* Request:  
+    .  
+    ```bash
+    curl 'localhost:5000/think/?subject=technology&subject=foo_bar'
+    ```
+
+* Response:  
+    .  
+    ```json
+    {
+        "technology": {
+            "error": false,
+            "quote": "For a successful technology, reality must take precedence over public relations, for Nature cannot be fooled.",
+            "author": "Richard Feynman"
+        },
+        "foo_bar": {
+            "error": true,
+            "reason": "Subject unknown"
+        }
+    }
+    ```
+
+<!---
+[Find the complete demo project on our github]()
+
+We need to opensource workshop code
+-->
+
+### Code used
 
-APM is available as part of the Datadog Agent with versions 5.11+ as part of the one line install for the Linux and Docker Agents. Currently, [Mac](https://github.com/DataDog/datadog-trace-agent#run-on-osx) and [Windows](https://github.com/DataDog/datadog-trace-agent#run-on-windows) users must perform a manual install of the APM Agent (aka Trace Agent) via a separate install process.
+We have two modules:
+
+* **Thinker API**: Catches the user request and forward it to *
+**thinker-microservice**
+    .  
+    ```python
```

Comment:
Hmm, this content seems to be overlapped. I'll pull the branch, make a fix and push the changes.

<img width="860" alt="screen shot 2018-02-09 at 10 07 28 am" src="https://user-images.githubusercontent.com/8942499/36034224-5756e95e-0d81-11e8-8006-1cd5102b637d.png">

Reply:
@AndrewMcBurney you need to remove the language parameter to your code block in order to make it work.. we need to fix this.. @davidejones would you know what's happening here?

Reply:
@l0k0ms The list items are setup in a slightly odd way using a pseudo element and some positioning. Its caused some problems before I can take a look at this.

Reply:
@l0k0ms @AndrewMcBurney I've made a commit that should fix this issue. When the preview site builds can you take a look see if that is good? Also it might be worth checking some random other pages that have list items to make sure they are as expected.